ProcessesApi/V1/Infrastructure/JWT/ProcessStoppedEventConstants.cs

```diff
@@ -0,0 +1,10 @@
+namespace ProcessesApi.V1.Infrastructure.JWT
+{
+    public static class ProcessStoppedEventConstants
```

Comment:
Can we rename this to ProcessClosedEventConstants as suggested in Jira

Reply:
oops, was pretty sure it's named ProcessStopped :) Will fix